cordapp-example/contracts-kotlin/src/main/kotlin/com/example/contract/CashContract.kt

```diff
@@ -18,14 +18,25 @@ class CashContract : Contract {
 
         when (command.value) {
             is Commands.Create -> requireThat {
-                // everyone can print/crate cash... because of covi... crisis
-                val cashState = tx.inputsOfType<CashState>().single()
-                "Cash value must be positive" using (cashState.balance > 0)
+                "There should be no input." using tx.inputsOfType<CashState>().isEmpty()
+                // in this exercise I allow anyone to create cash
+                val cashState = tx.outputsOfType<CashState>().single()
```

Comment:
Before this `.single()`, you could make sure there is a single output, with a nice message, like you do for the `.signers`.